packages/react-components/components/label/tests/chromatic/label.chroma.jsx

```diff
@@ -46,76 +51,76 @@ stories()
+                     <Label size="large" icon={<CommunicationIcon />}>Notification Sent</Label>
+                     <Label size="large" icon={ <MagnifierIcon />} iconPosition="right">Notification Sent</Label>
                  </div>
                  <div className="flex flex-column items-start">
                      <Label tag>
                          <Label circular empty size="tiny" color="red" />
                              Group added
                      </Label>
-                     <Label tag>
+                     <Label tag button>
                          <Label circular empty size="tiny" color="red" />
                              Group added
                          <Button icon circular size="tiny" ghost>
-                             <CloseIcon24 className="w4 h4" />
+                             <CloseIcon24 />
                          </Button>
                      </Label>
-                     <Label tag>
+                     <Label tag button icon={<CommunicationIcon />} iconPosition="right">
                          <Label circular empty size="tiny" color="red" />
                              Group added
-                         <CommunicationIcon className="ml2 w6 h6" />
                      </Label>
                  </div>
                  <div className="flex flex-column items-start">
                      <Label tag size="small">
                          <Label circular empty size="tiny" color="red" />
                             Group added
                      </Label>
-                     <Label tag size="small">
+                     <Label tag size="small" button>
                          <Label circular empty size="tiny" color="red" />
                              Group added
                          <Button icon circular size="tiny" ghost>
-                             <CloseIcon24 className="w4 h4" />
+                             <CloseIcon24 />
                          </Button>
                      </Label>
-                     <Label tag size="small">
+                     <Label tag size="small" icon={<CommunicationIcon />} iconPosition="right" >
                          <Label circular empty size="tiny" color="red" />
                              Group added
-                         <CommunicationIcon className="ml2 w6 h6" />
                      </Label>
                  </div>
                  <div className="flex flex-column items-start">
                      <Label tag size="large">
                          <Label circular empty size="tiny" color="red" />
                              Group added
                      </Label>
-                     <Label tag size="large">
+                     <Label tag size="large" button>
                          <Label circular empty size="tiny" color="red" />
                              Group added
                          <Button icon circular size="tiny" ghost>
```

Comment:
Oh and BTW, when I hover on this ghost button, visually nothing change, bug?

Reply:
That's because it's on a gray background ;)